setup.py

```diff
@@ -13,8 +13,10 @@
 import platform
 import shutil
 import sys
+import pandas as pd
```

Comment:
cool
Don’t use a for loop with an index (or counter) variable if you can replace it with the enhanced for loop (since Java 5) or forEach (since Java 8). It’s because the index variable is error-prone, as we may al...

📄 [Codejava](https://www.codejava.net/coding/10-java-core-best-practices-every-java-programmer-should-know#LoopIndex)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix